.github/workflows/verify_library.yml

```diff
@@ -180,7 +180,8 @@ jobs:
           service-account: ${{ secrets.CI_SERVICE_ACCOUNT }}
 
       - name: Run Storybook and upload preview
-        if: inputs.ENABLE_VISUAL_TESTING
+        # Skip running of Percy through storybook if PR is dependecy updates or if PR is draft
+        if: inputs.ENABLE_VISUAL_TESTING && github.actor != 'renovate[bot]' && (github.event_name == 'merge_group' || github.event.pull_request.draft != true)
```

Comment:
This could cause issues with the comment preview links being run when it shouldn't be if ENABLE_PLAYWRIGHT is not set to true - I'll attempt addressing that by attempting to have comment links step check for output of this step or playwright step